app/ledger.js

```diff
@@ -338,7 +338,7 @@ var reset = (doneP) => {
   if (!doneP) files.push(statePath)
   files.forEach((file) => {
     fs.unlink(pathName(file), (err) => {
-      if ((err) && (err.code !== 'ENOENT')) console.log(err)
+      if ((err) && (err.code !== 'ENOENT')) console.error(err)
```

Comment:
It might be helpful to also add in where it failed.  For example: `console.error('Error when deleting file "' + file + '": ', err)`